submissions/Valkirin/pop-up/index.html

```diff
@@ -0,0 +1,149 @@
+                    <span class="icon">
+                      <img src="ico/money.svg" alt="Money" />
+                    </span>
+                    <span class="icon_text">Money</span>
+                  </a>
+                </li>
+                <li class="content_item hide">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/count.svg" alt="Count" />
+                    </span>
+                    <span class="icon_text">Count</span>
+                  </a>
+                </li>
+                <li class="content_item hide">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/card.svg" alt="Pay" />
+                    </span>
+                    <span class="icon_text">Pay</span>
+                  </a>
+                </li>
+                <li class="content_item hide">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/menu.svg" alt="Quality" />
+                    </span>
+                    <span class="icon_text">Quality</span>
+                  </a>
+                </li>
+                <li class="content_item hide">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/folder.svg" alt="Folder" />
+                    </span>
+                    <span class="icon_text">Folder</span>
+                  </a>
+                </li>
+                <li class="content_item hide">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/map.svg" alt="Location" />
+                    </span>
+                    <span class="icon_text">Location</span>
+                  </a>
+                </li>
+                <li class="content_item hide">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/van.svg" alt="Delivery" />
+                    </span>
+                    <span class="icon_text">Delivery</span>
+                  </a>
+                </li>
+                <li class="content_item hide">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/woman.svg" alt="Woman" />
+                    </span>
+                    <span class="icon_text">Woman</span>
+                  </a>
+                </li>
+                <li class="content_item hide">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/man.svg" alt="Man" />
+                    </span>
+                    <span class="icon_text">Man</span>
+                  </a>
+                </li>
+              </ul>
+            </div>
+            <span class="arrow"></span>
```

Comment:
1. `arrow` logically is a part of `box`. You can put it inside `box` and calculate arrow positioning according `box` container current placement.
2. Instead of using one more element like `<span class="arrow"></span>` you can use pseudo-elements (`::before` or `::after`) on `box` element. Your markup will be more clean and visual part will be separated from document content.

Reply:
thank you for your comment